letsencrypt/Dockerfile

```diff
@@ -1,15 +1,10 @@
 FROM {{ hail_ubuntu_image.image }}
 
-RUN apt-get update -y
-
 # get add-apt-repository
-RUN apt-get update -y && \
-    apt-get install -y nginx software-properties-common && \
+RUN hail-apt-get-install nginx software-properties-common && \
     bash -c 'DEBIAN_FRONTEND=noninteractive apt-get install -y tzdata' && \
```

Comment:
Should we have not converted this image to be based on Ubuntu? Or was it always on ubuntu/base_image. Looking at the DEBIAN_FRONTEND variable

Reply:
Ubuntu is built on top of Debian, so it leaks into ubuntu things. Here, we're telling apt-get, a Debian package manager that Ubuntu adopted, to be non-interactive.

This was originally using the ubuntu image.